fms/models/granite.py

```diff
@@ -366,7 +366,7 @@ def forward(
         position_ids: Optional[torch.LongTensor] = None,
         past_key_value_states: Optional[Tuple[torch.FloatTensor,]] = None,
         use_cache: bool = False,
-        only_last_token: bool = False,
+        last_n_tokens: int = 0,
```

Comment:
This is a breaking api change that would require a 2.0 bump.
If we keep the `only_last_token` param, and handle it like
```
if only_last_token:
    assert last_n_tokens == 0, "Cannot set both last_n_tokens and only_last_token"
    last_n_tokens = 1
```

then we won't require a major version bump, and callers will not need to gate changes on a specific fms version